### ISSUE TITLE
Defining the viewbox property with the SVG directive

### ISSUE THREAD

**Issue by Vechro** [mint-lang/mint#318]
One of the only things you can't define in CSS for SVG is the `viewbox` property. I'd like to request a feature to pass a (perhaps optional) argument to the @svg directive that lets you define the `viewbox` size.

**Comment by s0kil** [comment]
We could also allow overwriting, the `width` and `height` attributes: https://github.com/mint-lang/mint/blob/master/src/compilers/directives/svg.cr#L18

**Comment by s0kil** [comment]
Are those three the most important, or should we also add all possible `<svg>` attributes?
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/svg#Attributes

**Comment by gdotdesign** [comment]
> Are those three the most important, or should we add all possible `<svg>` attributes?

Yes we can add other attributes as well, it might come in handy for some use cases.

**Comment by s0kil** [comment]
We currently [raise an issue](https://github.com/mint-lang/mint/blob/master/src/type_checkers/directives/svg.cr#L39-L41), when the `width` and `height` attributes are not defined on the SVG, is this required?

**Comment by gdotdesign** [comment]
> We currently [raise an issue](https://github.com/mint-lang/mint/blob/master/src/type_checkers/directives/svg.cr?rgh-link-date=2020-09-21T12%3A21%3A21Z#L39-L41), when the `width`, `height`, `viewBox` attributes are not defined on the SVG, is this required?

With the current implementation yes they are required [because we use them](https://github.com/mint-lang/mint/blob/master/src/compilers/directives/svg.cr#L18).

I don't see any reason why we could not make them optional (only use them when they are present), and overridable.

**Comment by s0kil** [comment]
```mint
const ALERT = @svg(../public/tiger.svg,
  preserveAspectRatio = "xMidYMid meet",
  viewBox = "0 0 100 100",
  width = 100,
  height = 100,
  x = 0,
  y = 0)
```
